weaviate/gql/get.py

```diff
@@ -181,7 +193,7 @@ def __init__(self, class_name: str, properties: Optional[PROPERTIES], connection
         self._limit: Optional[int] = None  # To store the limit filter if it is added
         self._offset: Optional[str] = None  # To store the offset filter if it is added
         self._after: Optional[str] = None  # To store the offset filter if it is added
-        self._near_ask: Optional[Filter] = None  # To store the `near`/`ask` clause if it is added
+        self._near_media: Optional[Filter] = None  # To store the `near`/`ask` clause if it is added
```

Comment:
Would it be possible to rename this to something more inclusive of the `NearText` and `NearAsk` clauses also? If I were coming to the code for the first time, I might expect `_near_media` to only be relevant to the `NearImage`, `NearAudio`, _etc._ clauses. Potentially just `self._near_clause` or `self._near_filter`?